multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2004,6 +2005,142 @@ def save(self, path: str, standalone: Optional[bool] = True):
                     f"removed."
                 )
 
+    def export_onnx(
+        self,
+        onnx_path: Optional[str] = None,
+        data: Optional[pd.DataFrame] = None,
+        batch: Optional[Dict] = None,
```

Comment:
Do we need the `batch` argument? Having the `data` argument seems enough. Having both could cause confusions and complex the logic.

Reply:
Removed. It was used for development